WordPress/Classes/Models/Comment.m

```diff
@@ -138,10 +138,16 @@ - (NSString *)authorUrlForDisplay
     return self.author_url.hostname;
 }
 
-- (BOOL)hasAuthorUrl {
+- (BOOL)hasAuthorUrl
+{
     return self.author_url && ![self.author_url isEqualToString:@""];
 }
 
+- (BOOL)isApproved
+{
+    return [self.status isEqualToString:@"approve"];
```

Comment:
Could you use `CommentStatusApproved` instead of hardcoding the string? In fact, there are a couple of other places in this file that use literals instead of those constants – might be worth updating them all!